packages/ui-extensions/docs/surfaces/point-of-sale/reference/apis/draft-order-api.doc.ts

```diff
@@ -0,0 +1,39 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+import {ExtensionTargetType} from '../types/ExtensionTargetType';
+import {generateCodeBlock} from '../helpers/generateCodeBlock';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'Draft Order API',
+  description:
+    'The Draft Order API provides an extension with data about the current draft order.',
+  isVisualComponent: false,
+  type: 'APIs',
+  requires:
+    ExtensionTargetType.PosPurchasePostActionMenuItemRender ||
+    ExtensionTargetType.PosPurchasePostActionRender ||
```

Comment:
I believe we're only adding the DraftOrderApi to `PosDraftOrderDetailsActionMenuItemRender` and `PosDraftOrderDetailsActionRender`

```suggestion
```